### PR TITLE
chore: Removed unused website.

### PR DESCRIPTION
**Pull request by ShogunPanda**
Removed unused website since we are moving to Lyra docs.

### CODE REVIEW

**Review by Ceres6 — COMMENTED** [review]
It seems lock file got outdated and the ci is complaining

Apart from the two comments in the files I the rest LGTM!

File: README.md [hunk 1]
```diff
@@ -30,22 +30,19 @@ be found in `./website` with the plugin configured.
 
 To test running the plugin locally run:
 
-```bash
```
[the remark this answers is not in the document]
Comment (ShogunPanda) [reply README.md:33]: Removed.

File: package.json [hunk 2]
```diff
@@ -1,53 +1,66 @@
...
+  "version": "0.0.0-beta.4",
+  "description": "Docusaurus plugin for local search powered by lyra",
+  "main": "lib/server/index.js",
+  "scripts": {
+    "build": "rm -rf lib && npm run build:client && npm run build:server && cp ./src/client/theme/SearchBar/search.css ./lib/client/theme/SearchBar/search.css && cp ./src/client/theme/SearchBar/styles.module.css ./lib/client/theme/SearchBar/styles.module.css",
+    "build:client": "tsc --project tsconfig.client.json ",
+    "build:server": "tsc --project tsconfig.server.json",
+    "watch:client": "tsc --watch --project tsconfig.client.json",
+    "watch:server": "tsc --watch --project tsconfig.server.json",
+    "test": ""
```
[the remark this answers is not in the document]
Comment (ShogunPanda) [reply package.json]: Changed.

File: package.json [hunk 3]
```diff
@@ -1,53 +1,68 @@
...
+    "lyra",
+    "docusaurus"
+  ],
+  "author": "",
   "license": "Apache-2.0",
   "bugs": {
     "url": "https://github.com/lyrasearch/plugin-docusaurus/issues"
   },
   "homepage": "https://github.com/lyrasearch/plugin-docusaurus#readme",
   "devDependencies": {
```
Comment (Ceres6) [inline package.json:37]: dev dependencies should be reinstalled and lint-staged config and scripts for prepare and build recovered

Comment (ShogunPanda) [reply package.json:38]: Yes, just noticed that. Done.

**Comment by ShogunPanda** [comment]
@Ceres6 All went good. Can you please approve?

**Review by Ceres6 — APPROVED** [review]
LGTM!

**Pull request merged**